control_msgs/msg/SingleDOFState.msg

```diff
@@ -0,0 +1,26 @@
+# This message presents current controller state of one degree of freedom.
+
+# DoF name, e.g., joint or Cartesian axis name
+string name
+
+# The set point, that is, desired state.
+float64 set_point
+
+# Current value of the process (ie: latest sensor measurement on the controlled value).
+float64 process_value
+
+# First time-derivative of the process value.
+float64 process_value_dot
+
+# The error of the controlled value, essentially process_value - set_point (for a regular PID implementation).
+float64 error
+
+# First time-derivative of the error of the controlled value.
+float64 error_dot
+
+# Time between two consecutive updates/execution of the control law.
+float64 time_step
+
+# Current output of the controller.
+float64 command
```

Comment:
According to ros-controls/roadmap#57 the naming should be the following:

@christophfroehlich or do you like more the old naming (then we can adjust ros-controls/roadmap#57

```suggestion
float64 reference

# Current value of the process (ie: latest sensor measurement on the controlled value).
float64 feedback

# First time-derivative of the process value. E.g., velcoity.
float64 feedback_dot

# The error of the controlled value, essentially process_value - set_point (for a regular PID implementation).
float64 error

# First time-derivative of the error of the controlled value.
float64 error_dot

# Time between two consecutive updates/execution of the control law.
float64 time_step

# Current output of the controller.
float64 output
```